userguide/tutorials/aws.adoc

```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
+
+**AWS Errors**
+
+Typical Issues are:
+  * Insufficient IAM Permissions
+  * Limit Exceeded
+  * Security Group Does Not Exist in VPC
+  * RDS Cluster failed to come up
+  * ...
+
+Make sure to check our `Pre-requisites` section to ensure you have setup a valid VPC with valid subnets. Fow AWS specific issues, please refer to the https://docs.amazonaws.cn/en_us/AWSCloudFormation/latest/UserGuide/troubleshooting.html[AWS troubleshooting documentation]
+    
+**Service Unavailable**
+
+We suggest to check the following:
+
+1. Is there any issues reported in the CFN logs
+2. Is the database up and running and accessible from the Kill Bill/Kaui ec2 instances?
+3. Is the database schema correctly installed?
+4. Is the Kill Bill/Kaui server correctly started and listening to the port `8080`?
+5. Are Kill Bill/Kaui server accessible from the LB, respectively on port `80` and `9090`?
+6. Are there any errors, stack traces in our logs
+
+==== Practical Tips
+
+**SSH to EC2 Instances**
+
+In order to answer these questions, you will first need to be able to SHH to the EC2 instances:
+
+From the EC2 dashboard, you can locate the instances by filtering on a prefix of the name of your stack, in my case `my-stack`,
+and as indicated below you will see the instances for KIll Bill server and KAUI. In the example below we see one of each:
+
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ec2-instances.png[align=center]
+
+You can select one instance and and then from the description tag, you will have access to:
+1. Public DNS
+2. The security group
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ec2-description.png[align=center]
+
+You will need to first click on the security group link to open the inbound port 22 required for SHH, as shown below:
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/security-group.png[align=center]
+
+Then, you can issue the SHH command, by copying the Public DNS from the description tab:
+
+```
+# SSH as ubuntu user
+> ssh -i  <LOCATION_KEY>/<KEY_NAME>.pem ubuntu@<PUBLIC_DNS>
+# Move to tomcat user
+> sudo su - tomcat
+```
+
+**CFN LOGS**
+
+
+If there are any issue with CFN, it should be available from `/var/log/cfn-init-cmd.log` or `/var/log/cfn-init.log`
+
+Other logs of interest may include:
+```
+/var/log/cfn-wire.log
+/var/log/cloud-init-output.log
+/var/log/awslogs.logs
+/var/log/xray/xray.log
+```
+
+**Kill Bill/KAUI Server Logs**
+
+The Kill Bill/KAUI server logs are located under `/var/lib/tomcat/logs/`, with the main 2 interesting logs being:
+* `killbill.out`: All Kill Bill server logs, configured as `INFO` by default
+* `kaui.out`: All KAUI server logs, configured as `INFO` by default
+* `localhost_access_log...`: Access requests to the severs
+
+The configuration of the logging (log rotation, log level, ...) can be found in `/var/lib/killbill/config/logback.xml`
```

Comment:
Maybe specify that updating that file requires a Tomcat restart (and specify the `service` commands)?